wiki/test-case-triggered-event-aggregation.md

```diff
@@ -1,7 +1,8 @@
-# TestCaseTriggeredEvent Aggregation
+# TestCaseTriggeredEvent aggregation
```

Comment:
Mattias commented earlier that titles should be capitilized, does the same apply here?